docs/templates/explanation-template.md

```diff
@@ -0,0 +1,52 @@
+---
+title: About <noun or nouns describing your subject> 
```

Comment:
@lena-larionova It was just something I thought of after looking at docs at other companies. GitHub has a similar rule for their explanation docs. I wasn't sure if a title like "About upstreams" would convey more clearly that the doc was explaining the topic vs. "Upstreams". 

I think the problem is more so that we sometimes call a tutorial or how-to topic a noun phrase like "Upstreams", so that makes it more confusing about what is conceptual info vs. task-based info.

Reply:
Hmm. I could see this being useful to have instead of calling everything "overview".

@Guaris any thoughts?

Reply:
I like it. I think we should do it. I think it may not be non-native english friendly, but that's only a gut feeling.